### PR TITLE
Installed handlebars

### PR DESCRIPTION
**Pull request by phiden**

### CODE REVIEW

**Comment by mapsam** [comment]
:hand: :bar_chart:

**Comment by bsmithgall** [comment]
Project idea -- glossary to convert @mapsam emoji to english

**Comment by mapsam** [comment]
Hand, bar!

File: src/handlebars/index.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- handlebars header -->
+{{> header }}
+
+	<div id="container">
+		
+		<section id="main">
+
```
Comment (bsmithgall) [inline src/handlebars/index.hbs]: Shouldn't this be `{{ content }}` or something?

Comment (phiden) [inline src/handlebars/index.hbs]: I was thinking we’d leave that part for mobbing?

Comment (bsmithgall) [inline src/handlebars/index.hbs]: +1

**Comment by bsmithgall** [comment]
@creade what additional test cases are needed?

**Comment by bsmithgall** [comment]
@mapsam @creade @phiden @ErnieAtLYD 

I think that we should go ahead and think about merging this sucker. Thinking more about the engine/ui functionality, these are super related. The thinking now is more along the following:

**UI**: @phiden will be leading the charge on wireframing, and taking the super basic `pickOne` functionality as moving it to a really nice thing.

**Engine**: I want to take a crack at implementing the external API. @ErnieAtLYD if you want to work on this too, this would be fun fun fun. The realization here, though, is that triggers need to have both (super simple) templates and functionality handling.

This changes the work around a bit, but not overmuch. I think that the workflow (for now) should be:
1. A trigger is added
2. The trigger is fixed up with styling

This basically requires the changes that we are making in this branch. NOTE: we can always drop the handlebar dependency at a later date.

Thoughts?

**Comment by phiden** [comment]
Sounds good. Here's what I'm thinking on the UI end:
- classes should all carry the prefix `Wand-` to avoid conflicts
- we could offer skins that have various color stories and fonts 
- eventually, the editor will need to be considered as well
- wireframes and such can provide guidelines 
- @TiffanyAndrews's going to teach us about CSS testing!

**Comment by ErnieAtLYD** [comment]
> ...Triggers need to have both (super simple) templates and functionality handling. This changes the work around a bit, but not overmuch. I think that the workflow (for now) should be: 1) A trigger is added and 2) The trigger is fixed up with styling.

@bsmithgall: I'm okay with this, but I may want to follow your lead as it sounds like you have a definite thought process for laying out the module structures. Is that correct? Does it make sense to commit the beginnings of a framework to the `engine` branch?

**Comment by ErnieAtLYD** [comment]
Oh hey it looks like `wand.engine` has already been implemented! I should have checked the code first. Looks good to me :+1:

**Comment by mapsam** [comment]
YES @TiffanyAndrews I would love to see some CSS testing! :facepunch:

**Pull request merged**